x-pack/plugins/apm/public/components/shared/span_links/span_links_table.tsx

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  EuiBasicTableColumn,
+  EuiButtonEmpty,
+  EuiButtonIcon,
+  EuiCopy,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiInMemoryTable,
+  EuiLink,
+  EuiPopover,
+  EuiText,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useState } from 'react';
+import { SpanLinkDetails } from '../../../../common/span_links';
+import { asDuration } from '../../../../common/utils/formatters';
+import { useApmParams } from '../../../hooks/use_apm_params';
+import { useApmRouter } from '../../../hooks/use_apm_router';
+import { ServiceLink } from '../service_link';
+import { getSpanIcon } from '../span_icon/get_span_icon';
+
+interface Props {
+  items: SpanLinkDetails[];
+}
+
+export function SpanLinksTable({ items }: Props) {
+  const { link } = useApmRouter();
+  const {
+    query: { rangeFrom, rangeTo },
+  } = useApmParams('/services/{serviceName}/transactions/view');
+  const [idActionMenuOpen, setIdActionMenuOpen] = useState<
+    string | undefined
+  >();
+
+  const columns: Array<EuiBasicTableColumn<SpanLinkDetails>> = [
+    {
+      field: 'serviceName',
+      name: i18n.translate('xpack.apm.spanLinks.table.serviceName', {
+        defaultMessage: 'Service name',
+      }),
+      sortable: true,
+      render: (_, { details }) => {
+        if (details) {
+          return (
+            <ServiceLink
+              serviceName={details.serviceName}
+              agentName={details.agentName}
+              query={{
+                rangeFrom,
+                rangeTo,
+                kuery: '',
+                serviceGroup: '',
+                comparisonEnabled: true,
```

Comment:
you need to use the kibana advanced setting here no?

Reply:
yes, I do, thanks